experimental/PartitionedPermutations/README.md

```diff
@@ -0,0 +1,33 @@
+# Partitioned Permutations
+
+This project implements algorithms for working with partitioned permutations. These are used in the context of Free Probability Theory to model higher order freeness and higher order free cumulants. The central reference is the paper [Second Order Freeness and Fluctuations of Random Matrices III. Higher order freeness and free cumulants (2007)](https://arxiv.org/pdf/math/0606431.pdf).
```

Comment:
please use citations using the bibliography. see [docs.oscar-system.org/dev/DeveloperDocumentation/documentation/#Updating-the-bibliography](https://docs.oscar-system.org/dev/DeveloperDocumentation/documentation/#Updating-the-bibliography)

Reply:
has been addressed